src/Workspaces/Remote/Core/Services/AssetStorage.cs

```diff
@@ -18,8 +18,8 @@ internal class AssetStorage
     {
         public static readonly AssetStorage Default = new AssetStorage();
 
-        private const int CleanupInterval = 3; // 3 minutes
-        private const int PurgeAfter = 30; // 30 minutes
+        private const int CleanupInterval = 1; // 1 minutes
+        private const int PurgeAfter = 3; // 3 minutes
```

Comment:
now these are just temp storage until solution is updated. primary workspace itself is now the one that hold onto data.